pkg/registry/core/service/ipallocator/controller/repair.go

```diff
@@ -67,6 +67,7 @@ func NewRepair(interval time.Duration, registry service.Registry, network *net.I
 // RunUntil starts the controller until the provided ch is closed.
 func (c *Repair) RunUntil(ch chan struct{}) {
 	wait.Until(func() {
+		time.Sleep(5*time.Second)
```

Comment:
Will be removed - was here to proove my hypothesis and this is the reason of flakes.